src/GSvar/IGVInitCachePhaseOneWorker.h

```diff
@@ -0,0 +1,24 @@
+#ifndef IGVINITCACHEPHASEONEWORKER_H
+#define IGVINITCACHEWORKER_H
+
+#include <QRunnable>
+#include "IGVInitCache.h"
+#include "VariantList.h"
+
+// This worker requests FileLocation information from the server using FileLocationProvider,
+// it is started when a user opens a sample to reduce a delay for the initial IGV call
+class IGVInitCachePhaseOneWorker
```

Comment:
How long do the two workers take?
If we had only one worker, no concurrency could occur and the code could be much easier because we do not need the IGVCacheList class...